thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/completeness/checker/DataCompletenessResource.java

```diff
@@ -0,0 +1,58 @@
+package com.linkedin.thirdeye.completeness.checker;
```

Comment:
I think all rest resources are part of that package for now. We haven't done a good job on separate needed for dashboard v/s not. I don't know if we have many such cases. For now, keeping all rest resources in one place is probably the best solution for now.